matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/util/ContentUtils.kt

```diff
@@ -44,4 +46,15 @@ object ContentUtils {
         }
         return repliedBody
     }
+
+    @Suppress("RegExpRedundantEscape")
+    fun formatSpoilerTextFromHtml(formattedBody: String): String {
```

Comment:
This would deserve some unit test...

Reply:
although it uses `androidx.core.text.HtmlCompat` so we wouldn't be able to write a unit test unless we use a java equivalent